vip_spec.xsd

```diff
@@ -337,8 +340,11 @@ reference elements for polling location
         <xs:element name="Person">
           <xs:complexType>
             <xs:sequence>
+              <!-- SURVEY: Types xs:string and InternationalizedText seem
+                   to be used inconsistently here. Suggestions?
```

Comment:
I'm just wondering why profession and title are internationalized but names, nicknames, etc, aren't.